package.json

```diff
@@ -26,7 +26,7 @@
     "localforage": "1.7.2",
     "localforage-observable": "1.4.0",
     "marked": "github:mattermost/marked#ed33baecd7d7fa97d479ba22dde9d226b083d67d",
-    "mattermost-redux": "github:mattermost/mattermost-redux#c3a7215403d8607a21a30e11f6066308c448ce75",
+    "mattermost-redux": "github:kkekkonen/mattermost-redux#mattermost-redux-rethreading",
```

Comment:
:warning: probably shouldn't be part of the PR

Reply:
I guess I should wait for the mm-redux PR to be merged first